### DISCUSSION TITLE
NCAA Rotamers for Cyclosporin

### DISCUSSION THREAD

**Discussion by Mikukub** [RosettaCommons/rosetta#242] [General]
Where can I file the ncaa_rotamer_libraries/cyclosporin/BMT_Nmethylated.rotlib?  
Do I need to create rotamers for each amino acid myself? If so, how can I do that? I am quite new to Rosetta.

**Comment by roccomoretti** [comment]
They do not seem to have been added to the database when the BMT.params file was added. They also do not seem to be part of the separate NCAA Rotamer Library download (obtainable from the release downloads pages from https://rosettacommons.org/software/download/ )

I don't know if @vmullig still has them kicking around and might be able to point you to a download location.
